### PR TITLE
Add solver classifier intern project description

### PR DESCRIPTION
**Pull request by bjoernh2000**
Passed my local pre-commit

### CODE REVIEW

**Comment by KPostOffice** [comment]
Hey, Bjoern. The reason your getting the `pre-commit` errors is because by default when pre-commit checks your files it only checks files which have been changed by the commit that you're checking. However, when we run the CI check on Git Hub we use `pre-commit run --all-files` which will check all files present in the repo.  For the most part (if pre-commit is doing its job) then the two should end up with the same output, sometimes they don't and you need to run `pre-commit run --all-files` yourself.

**Comment by bjoernh2000** [comment]
> Hey, Bjoern. The reason your getting the `pre-commit` errors is because by default when pre-commit checks your files it only checks files which have been changed by the commit that you're checking. However, when we run the CI check on Git Hub we use `pre-commit run --all-files` which will check all files present in the repo. For the most part (if pre-commit is doing its job) then the two should end up with the same output, sometimes they don't and you need to run `pre-commit run --all-files` yourself.
> 
> ping me if you need any help

Hey Kevin,

Thanks a lot that made more sense, I was confused because the file that was failing the pre-commit was on thoth-station master branch unedited, but I used the local check to run the fix on my fork.

**Comment by goern** [comment]
@bjoernh2000 the PR title/body is a little miss-leading, otherwise

/lgtm 
/assign @fridex 
/unassign @goern

**Review by fridex — APPROVED** [review]
lgtm, thanks! 👍🏻 

Sounds like it will be fun 😉

File: docs/intern-projects/classify-errors-produced-during-installation-of-python-modules.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+* Access to Red Hat network
+* Access to GMail and Google Chat
+    * Be part of Thoth-Station Google Chat Room
+* Be part of Thoth scrum meetings
+    * You should receive an invite to a Google calendar Event
+* Access to GitHub using your GitHub your account
+    * Thoth Station
+* Get access to JupyterHub
+    * https://jupyterhub.datahub.redhat.com/
```
Comment (goern) [inline docs/intern-projects/classify-errors-produced-during-installation-of-python-modules.md:23]: https://odh.operate-first.cloud/

**Pull request merged**